### DISCUSSION TITLE
Tool head weight

### DISCUSSION THREAD

**Discussion by foliba** [bassamanator/everything-sovol-sv06#41] [General]
Does anyone know how much the std. tool head's weight is?

**Comment by bassamanator** [comment]
You could always unmount it from the printer and weigh.

**Reply by foliba** [reply]
If I had an unmodified tool head around. :-D

**Reply by bassamanator** [reply]
I'll measure mine next time I disassemble, could be months away. Hopefully someone else will chime in.

**Comment by bassamanator** [comment]
### SV06

The entire hotend assembly weighs 384g.

I'm using an aftermarket heatblock that weighs 24g.
